.env.docker.production

```
@@ -0,0 +1,19 @@
+# Docker Environment Configuration for production
+# Generated on 2025-09-12T22:47:44.340Z
+
+NODE_ENV=production
+PORT=3000
+DATABASE_URL=postgres://binaai_user:${DB_PASSWORD}@${DB_HOST}:5432/binaai_prod
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the NODE_ENV key

(UnorderedKey)